internal/spot/spotinst_services_ocean_k8s_aws.go

```diff
@@ -453,6 +453,12 @@ func (x *oceanKubernetesAWSClusterBuilder) buildStrategy() *aws.Strategy {
 		strategy.SetDrainingTimeout(spotinst.Int(x.opts.DrainingTimeout))
 	}
 
+	if x.fs.Changed(flags.FlagOceanAvailabilityVsCost) {
+		test := aws.ClusterOrientation{}
```

Comment:
@yehielnetapp 
Please change the name test into something more suitable like: clusterOrientation